README.md

```diff
@@ -29,6 +29,7 @@ module "terraform-dome9-awp-aws" {
   # e.g:
   awp_cross_account_role_name        = "<CrossAccountRoleName>"
   awp_cross_account_role_external_id = "<ExternalId>"
+  awp_centralized_cloud_account_id   = "in case of centralized onboarding, set the 'AWS account id' of the central account where the scans executed"
   awp_additional_tags                = {}  # e.g {"key1" = "value1", "key2" = "value2"}
     
 
```

Comment:
If settings are not allowed in centralized/sub, please write it (and hopefully you validated it at the api)

Reply:
done